libraries/AP_RSSI/AP_RSSI.cpp

```diff
@@ -187,8 +187,12 @@ float AP_RSSI::read_pin_rssi()
 // read the RSSI value from a PWM value on a RC channel
 float AP_RSSI::read_channel_rssi()
 {
```

Comment:
Shouldn't we fix this in the `read_receiver_rssi` function so it works with analog and PWM pin RSSI too? That would remove the need for the RCProtocol change for the time being too as I think this is the only user?

Reply:
I think that makes sense...  I updated the PR to do that.